core/domain/classifier_domain.py

```diff
@@ -262,7 +273,7 @@ def update_status(self, status):
                     initial_status, status))
         self._status = status
 
-    def to_dict(self):
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
@DubeySandeep hey, I have a doubt. I think I should change `Any` here but I am bothered about how to change it. should I use `TypedDict` to create a class then replace it with `Dict[str, Any]. I need your suggestion

Reply:
Can you please check other files and see how this is handled?

Reply:
Sure, will do that.

Reply:
yes, I did and I came to know how it is passed through different files. I am updating the PR. Please take a look. Thanks.

Reply:
done